CHANGELOG.md

```
@@ -7,6 +7,7 @@ This changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.
 ## [Unreleased]
 
 ### Added
+- Refactored token classes to use dataclass-driven `__str__ `and `__repr__` generation. [#999](https://github.com/hiero-ledger/hiero-sdk-python/issues/999)
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)